smsdk/client.py

```diff
@@ -371,9 +394,22 @@ def get_type_from_machine(self, machine_source=None, **kwargs):
             machine_source, **kwargs
         )
 
-    def get_machine_schema(
-        self, machine_source, types=[], show_hidden=False, return_mtype=False, **kwargs
-    ):
+    def get_machine_schema(self, *args, **kwargs):
```

Comment:
We could have set the args to default to None this leads to more readable code.

Reply:
'args' are option positional arguments. can't be set to None by default in the function signature.